packages/botonic-react/src/components/button.jsx

```diff
@@ -37,6 +37,7 @@ export const Button = props => {
     else if (props.path) sendPayload(`__PATH_PAYLOAD__${props.path}`)
     else if (props.payload) sendPayload(props.payload)
```

Comment:
@ericmarcos if we allowed having both payload and url, would we be able to open an url and send a track when user presses a button?